### PR TITLE
Ams 5 full registration

### PR DESCRIPTION
**Pull request by alex-parkhamovich**
Full registration with 2 steps, change controllers actions to defaults show/update and etc.

### CODE REVIEW

File: app/controllers/doctor_admin/after_signup_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  def show
+    @personal = current_user.doctor_personal
+    render_wizard
+  end
+
+  def update 
+    @personal = current_user.doctor_personal
+    status = 'completed'
+    if @personal.update_attributes(doctor_personal_params)
```
Comment (tuwukee) [inline app/controllers/doctor_admin/after_signup_controller.rb]: Why don't you handle errors on update?

File: app/controllers/patient_admin/after_signup_controller.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  steps :personal
+
+  layout 'patient_auth'
+
+  def show
+    @personal = current_user.patient_personal
+    render_wizard
+  end
+
+  def update
```
Comment (tuwukee) [inline app/controllers/patient_admin/after_signup_controller.rb]: A lot of code duplication, is it possible to inherit it from some base controller with list of the default actions or store shred methods in the concerns?

File: app/controllers/patient_admin/after_signup_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  layout 'patient_auth'
+
+  def show
+    @personal = current_user.patient_personal
+    render_wizard
+  end
+
+  def update 
+    @personal = current_user.patient_personal
```
Comment (tuwukee) [inline app/controllers/patient_admin/after_signup_controller.rb]: Where is the rspec tests? >:)

**Review by tuwukee — CHANGES_REQUESTED** [review]

File: app/controllers/doctor_admin/after_signup_controller.rb (hunk 1 above)
Comment (alex-parkhamovich) [reply app/controllers/doctor_admin/after_signup_controller.rb]: forgot it, i'll add it in next commit.

File: app/controllers/patient_admin/after_signup_controller.rb (hunk 3 above)
Comment (alex-parkhamovich) [inline app/controllers/patient_admin/after_signup_controller.rb]: in progress ^^

**Review by tuwukee — APPROVED** [review]

**Pull request merged**